.github/workflows/buildimages-update.yml

```diff
@@ -133,19 +139,25 @@ jobs:
           GH_TOKEN: ${{ github.token }}
           PR_TITLE: "[automated] ${{ steps.update_build_images.outputs.MESSAGE }}"
           PR_LABELS: "go-update,team/agent-shared-components"
+          CURRENT_BUILDIMAGE_TAG: ${{ steps.current_buildimage_tag.outputs.BUILDIMAGE_TAG }}
```

Comment:
Out of curiosity, does this qualifies as untrusted output since it comes from our own tasks?

Reply:
True this should be fine but the linter (it's coming soon in the CI) that I'm using does not make any difference, so I fixed it as well !

Reply:
Fair enough :)